lib/SPIRV/libSPIRV/SPIRVStream.cpp

```diff
@@ -222,6 +222,37 @@ bool SPIRVDecoder::getWordCountAndOpCode() {
   return true;
 }
 
+bool SPIRVDecoder::getNextSPIRVWord(SPIRVWord &Word) {
+  if (IS.eof()) {
+    Word = -1;
+    SPIRVDBG(spvdbgs() << "[SPIRVDecoder] getNextSPIRVWord EOF " << Word
+                       << '\n');
+    return false;
+  }
+  *this >> Word;
+#ifdef _SPIRV_SUPPORT_TEXT_FMT
+  if (SPIRVUseTextFormat) {
```

Comment:
@pauzinl Could you please clarify why code on lines 235-241 duplicates code on lines 244-250?

Reply:
@mlychkov I thought it should work the same as `getWordCountAndOpCode`. Anyway, I will remove this code (233-243) because we don't need to work with text SPIR-V.